diagnostic.md

```diff
@@ -21,13 +21,20 @@ Write the `curl` request you'd use to retrieve a list of **all donuts** on the
 server.
 
 ```sh
-# your answer here
+curl "http://www.example.com/donuts" \
+--include  \
+--request GET \
```

Comment:
Should look more like this: `curl --include --request GET http://www.example.com/donuts`